examples/audio_decoder.rs

```diff
@@ -12,9 +12,11 @@ use std::path::Path;
 use std::sync::mpsc;
 use std::sync::{Arc, Mutex};
 use std::{thread, time};
+use servo_media::audio::muteable::MediaContextId;
 
 fn run_example(servo_media: Arc<ServoMedia>) {
-    let context = servo_media.create_audio_context(Default::default());
+    let context_mutex = servo_media.create_audio_context(MediaContextId::build(1, 1), Default::default());
```

Comment:
Same thing about the naming: `context`.

Reply:
could you elaborate on this case?
there is an actual `context`, which is locked & unwrapped from `context_mutex` at the next line; also i need to bind `context_mutex` since it needs to live till the end of `run_example`

would renaming `context_mutex` -> `context` and `context` -> `locked_context` be ok?

Reply:
There is no need to use a different variable name in this case. Rust provides a way to [shadow variables](https://doc.rust-lang.org/book/ch03-01-variables-and-mutability.html#shadowing).

```rust
    let context = servo_media.create_audio_context(MediaContextId::build(1, 1), Default::default());
    let context = context.lock().unwrap();
```